### PR TITLE
Cult desc changes

### PR DESCRIPTION
**Pull request by Dragonkiller93**
## What Does This PR Do
Changes cultist spell descriptions to be more helpful. Stun hand will actually tell you to stab now, EMP won't use the word "channel", which typically implies a delay instead of instantly going off, and conceal presence will now switch between revealing and concealing in its description after the first use.

## Why It's Good For The Game
This should help newer players, or anybody who reads the spell descriptions, so they can understand what their spells do. Especially relevant with the stun hand change.

## Changelog
:cl: Iwanabeu
tweak: Changes Cultist spell descriptions to be more helpful.
/:cl:

### CODE REVIEW

**Comment by Dragonkiller93** [comment]
I don't know how to remove that adds settings to gitignore commit, any help would be appreciated.

File: code/game/gamemodes/cult/blood_magic.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@
 
 /datum/action/innate/cult/blood_spell/equipment
 	name = "Summon Equipment"
-	desc = "Allows you to empower your hand to summon combat gear onto a cultist you touch, including cult armor, a cult bola, and a cult sword."
+	desc = "Empowers your hand to summon combat gear onto a cultist you touch, including cult armor (if not wearing a jacket), a cult bola, and a cult sword."
```
Comment (Sirryan2002) [inline code/game/gamemodes/cult/blood_magic.dm:238]:
```suggestion
	desc = "Empowers your hand to summon combat gear onto a cultist you touch, including cult armor into open slots, a cult bola, and a cult sword."
```

File: code/game/gamemodes/cult/blood_magic.dm [hunk 2]
```diff
@@ -339,7 +339,7 @@
 		button_icon_state = "veiling"
 	if(charges <= 0)
 		qdel(src)
-	desc = base_desc
+	desc = (revealing? "Reveals" : "Conceals")+ " nearby cult structures, airlocks, and runes."
```
Comment (Sirryan2002) [inline code/game/gamemodes/cult/blood_magic.dm:342]:
```suggestion
	desc = "[revealing ? "Reveals" : "Conceals"] nearby cult structures, airlocks, and runes."
```

**Comment by Dragonkiller93** [comment]
I think all the other stuff is fixed, only files changed are the intended ones.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**